cap-ci/create_pipeline.sh

```diff
@@ -54,7 +53,7 @@ fly_args=(
 )
 
 # space-separated paths to template files and directories which contain template files
-template_paths="${pipeline_config} scripts jobs common"
+template_paths="scripts jobs common"
 templates=$(find ${template_paths} -type f -exec echo "--template="{} \;)
 pipeline_file=${3:-pipeline.yaml.tmpl}
-fly "${fly_args[@]}" --config <(gomplate --verbose ${templates} --file "${pipeline_file}")
+fly "${fly_args[@]}" --config <(gomplate --verbose --datasource config="${pipeline_config}" ${templates} --file "${pipeline_file}")
```

Comment:
I'm on the side of having them as config files.

Templating common parts between the 2 configs means that changing the implementation in the future becomes more difficult, because one needs to undo the templating to be able to fly your own development pipeline.

While I see the point on templating so there's no repetition of config options and we don't drift there, I think that another solution for that is just to automate the flying of the pipelines when master changes. I would prefer doing that, which solves other problems in addition, and allows us to leave the code as simple as possible for touch-and-go contributors.